packages/picasso-forms/src/Form/Form.tsx

```diff
@@ -38,6 +38,21 @@ const setActiveFieldTouched = <
   }
 }
 
+const setHasMultilineCounter = <
+  FormValues = object,
+  InitialFormValues = Partial<FormValues>
+>(
+  args: any[],
```

Comment:
maybe we can type these args, wdyt? We know the first one is a name and the second is a a boolean, makes sense to type them like this?
```suggestion
  args: [name: string, hasCounter: boolean],
```

Reply:
Good idea 👍 done ✅